docs/faq.rst

```diff
@@ -210,8 +210,25 @@ be used used to bind a socket locally::
     async with aiohttp.ClientSession(connector=conn) as session:
         ...
 
-.. seealso:: :class:`aiohttp.TCPConnector` and ``local_addr`` parameter.
+If your OS supports it, you may also bind directly to the interface ::
+
+    conn = aiohttp.TCPConnector(network_interface="eth0")
+    async with aiohttp.ClientSession(connector=conn) as session:
+        ...
+
+It is also possible to have a fallback by catching the ``RuntimeError`` on creation:
+
+    try:
+        conn = aiohttp.TCPConnector(network_interface="eth0")
+    except RuntimeError:
+        # fallback to local_addr as the runtime kernel is unsupported.
+        conn = aiohttp.TCPConnector(local_addr=("127.0.0.1", 0))
+
+    async with aiohttp.ClientSession(connector=conn) as session:
+        ...
```

Comment:
I'd merge two code examples into one and have a combined paragraph with common explanation.